org.sonatype.m2e.webby/src/org/sonatype/m2e/webby/internal/config/WarConfigurationExtractor.java

```diff
@@ -198,8 +198,7 @@ public WarConfiguration getConfiguration(IMavenProjectFacade mvnFacade, MavenPro
   private String resolve(String basedir, String path) {
     String result = path;
     if(path != null && basedir != null) {
-      path = path.replace('\\', File.separatorChar).replace('/', File.separatorChar);
-
+      path = path.replace('\\', '/');
```

Comment:
Why is this change necessary? The old code appears correct for both Windows and Unix filesystems.